ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/EthSigner.java

```diff
@@ -92,6 +96,41 @@ public void run() {
     runner.start();
   }
 
+  private HttpClientOptions applyConfigTlsSettingsTo(final HttpClientOptions input) {
+    final HttpClientOptions result = new HttpClientOptions(input);
+    boolean tlsIsRequired =
+        config.getWeb3TrustStoreOptions().isPresent()
+            || config.getClientCertificateOptions().isPresent();
+
+    if (tlsIsRequired) {
+      result.setSsl(true);
+      try {
+        if (config.getWeb3TrustStoreOptions().isPresent()) {
+          result.setPfxTrustOptions(convertFrom(config.getWeb3TrustStoreOptions().get()));
+        } else {
+          result.setTrustAll(true);
```

Comment:
is this deliberate? usually trustAll can be considered as security issue. If web3 trust store is present, use that, otherwise, use system defined/default trust store.